snake_cafe.py

```diff
@@ -79,42 +86,67 @@ def greeting():
 
     '''))
 
-question = dedent('''
-            **  Place order for another item  **\n
-            **  Type order to see your current order  **\n
-        ''')
+def start_order():
+    greeting()
+    item =  input('Would you like custom menu: ').lower()
+    if item == 'yes':
+        open_csv_menu()
+        place_order()
+    else:
+        print_menu()
+        place_order()
+
+
+def open_csv_menu():
+    '''opens custom menu as csv file. throws an error if its not csv '''
+    try:
+        with open('menu.csv') as csvfile:
+            readCSV = csv.reader(csvfile, delimiter=',')
+            line_count = 0
+            for row in readCSV:
+                if line_count == 0:
+                    print('***************************************')
+                    print(f'*******    {", ".join(row)}    *********')
+                    print('***************************************')
+                    line_count += 1
+                else:
+                    print(f'\t{row[0]} ** {row[1]} ** Qt({row[2]}) --------- {row[3]}$')
+                    line_count += 1
+    except (FileNotFoundError, TypeError)as e:
+            print(e)
+            
+            
 
 
-menu = {'wings' : 7, 'cookies': 1, 'taco': 2, 'nachos': 2, 'sliders':6, 'rolls': 2,'salmon': 5,'steak': 3, 'meat tornado': 4, 'cod': 20, 'pizza':9, 'green salad':9, 'potato salad': 7, 'Roasted Potatoes': 11,'mashed potatoes': 7,'coleslaw': 5, 'sausage': 6, 'ice cream': 3, 'cake': 2,'pie': 5, 'banana bread':4, 'sorbet':5, 'chocolate':3, 'coffee': 2, 'tea': 2, 'bloody mary': 10, 'Beer': 5, 'wine': 6, 'juice': 2  }
+
+menu = {'wings' : 7, 'cookies': 1, 'taco': 2, 'nachos': 2, 'sliders':6, 'rolls': 2,'salmon': 5,'steak': 10, 'meat tornado': 4, 'cod': 20, 'pizza':9, 'green salad':9, 'potato salad': 7, 'Roasted Potatoes': 11,'mashed potatoes': 7,'coleslaw': 5, 'sausage': 6, 'ice cream': 3, 'cake': 2,'pie': 5, 'banana bread':4, 'sorbet':5, 'chocolate':3, 'coffee': 2, 'tea': 2, 'bloody mary': 10, 'Beer': 5, 'wine': 6, 'juice': 2  }
 
 order = defaultdict(int)
 
 def place_order():
     order_limit = 0
-    greeting()
-    while order_limit < 20:
+    
+    while order_limit < 50:
         item =  input('Enter item (enter \'order\' to see you order): ').lower()
         if item in menu:
-            order[item] += 1
+            # enter item quantity
+            item_qt = input('Enter quantity: ')
```

Comment:
This is a good spot for some error handling. If I enter a string instead of an integer, your code crashes with a ValueError.